src/I18n/TranslatorRegistry.php

```diff
@@ -43,6 +43,25 @@ class TranslatorRegistry extends TranslatorLocator
     protected $_defaultFormatter = 'default';
 
     /**
+     * A CacheEngine object that is used to remember translator across
+     * requests.
+     *
+     * @var \Cake\Cache\CacheEngine
+     */
+    protected $cacher;
+
+    /**
+     * Sets the CacheEngine instance used to remember translators across
+     * requests.
+     *
+     * @param \Cake\Cache\CacheEngine $cacher The cacher instance.
+     * @return void
+     */
+    public function setCacher(CacheEngine $cacher) {
```

Comment:
Curly bracket should be on the next line in PSR

Not very fond of "cacher" for this (but yes, it's consistent with "translator")